pkg/performanceprofile/controller/performanceprofile/components/kubeletconfig/kubeletconfig.go

```diff
@@ -13,6 +13,8 @@ import (
 	performancev2 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/performanceprofile/v2"
 	"github.com/openshift/cluster-node-tuning-operator/pkg/performanceprofile/controller/performanceprofile/components"
 	machineconfigv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+
+	kubeletv1beta1 "k8s.io/kubernetes/pkg/kubelet/apis/config/v1beta1"
```

Comment:
is this the right version ? v1beta1?

Reply:
note we also have `v1` nowadays https://github.com/kubernetes/kubernetes/tree/master/pkg/kubelet/apis/config/v1

Reply:
there is a v1, but with no [defaults_test.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/config/v1beta1/defaults_test.go)

Reply:
Then the option is come back to the initial commit and hardcore the values.

Reply:
@fromanirh `v1` is not included under the 1.25 branch https://github.com/kubernetes/kubernetes/tree/release-1.25/pkg/kubelet/apis/config  so under kubelet context can we assume we need to use v1beta1 ?

Reply:
the problem here is painting ourselves in a corner. This can be indeed be fine for 1.25, but what next?
- we know we will have to maintain this code
- we know we will have to move away from v1beta1 eventually
so this is an improvemente for the short term, but what about mid/long term?

Reply:
Well my opinion it would be do it as in the initial commit even it is not the best option. Use v1beta1 seems as we introduce two technical debs, the hardcored values an the dependency of v1beta1.

Reply:
From my perspective replicating defaults is fine as long as we have a test which fails when our defaults drift away from kube's (because kube changes their defaults).

Reply:
Ok, I am updating with a test

Reply:
Add test: https://github.com/openshift/cluster-node-tuning-operator/blob/4a4875077e31778562e694916738c7a80c47a5af/pkg/performanceprofile/controller/performanceprofile/components/kubeletconfig/kubeletconfig_test.go#L158